README.md

```diff
@@ -78,11 +78,22 @@ integration into the CLI may also be deferred to a later date, or a later
 semver-major CLI release, at the npm collaborators' discretion. All the RFC does
 is communicate the team's consensus to accept a change.
 
+### Implementation
+
 When the changes described in an RFC have been implemented and merged into the
 relevant repository (and thus, due to be released), the corresponding RFC will
 be moved from `accepted/` to `implemented/`. If you'd like to implement an
 accepted RFC, please make a PR in the appropriate repo and mention the RFC in
-the PR. Feel free to do this even for work-in-progress code!
+the PR. Feel free to do this even for work-in-progress code.
+
+### Withdrawal
+
+From time to time npm collaborators will review RFCs awaiting implementation to
+ensure their accuracy and relevance. In cases where a previously ratified RFC is
+deemed to no longer be a viable candidate for implementation, an [**amendment
+section**](withdrawn/0000-template.md) will be added to the bottom of the
+document outlining the reason for repeal and subsequently moved to the
```

Comment:
My only nit is that the amendment section should be at the top, not the bottom, so that it's harder to miss.

Reply:
👍🏻 *Updated*